pkg/cmd/server/api/types.go

```diff
@@ -1407,3 +1408,16 @@ type DefaultAdmissionConfig struct {
 	// Disable turns off an admission plugin that is enabled by default.
 	Disable bool
 }
+
+// TemplateServiceBrokerConfig holds information related to the template
+// service broker
+type TemplateServiceBrokerConfig struct {
+	// Enabled switches on the types, API and controllers associated with the
+	// template service broker
+	Enabled bool
+
+	// TemplateNamespaces indicates the namespace(s) in which the template service
+	// broker looks for templates to serve.  If empty, the default is the
```

Comment:
I'm not sold on the default.  How about when you create one, you set that value, but you don't actually default it on the object?